classes/UpgradeTools/CoreUpgrader/CoreUpgrader.php

```diff
@@ -689,6 +701,75 @@ protected function updateTheme()
         }
     }
 
+    protected function updateRTLFiles()
+    {
+        if (!$this->container->getUpgradeConfiguration()->shouldUpdateRTLFiles()) {
+            return;
+        }
+
+        // BO theme
+        if (class_exists(RtlStylesheetProcessor::class)) {
+            $this->logger->info($this->container->getTranslator()->trans('Upgrade the RTL files of back-office themes.', [], 'Modules.Autoupgrade.Admin'));
+
+            $this->removeExistingRTLFiles([
+                ['directory' => $this->container->getProperty(UpgradeContainer::PS_ADMIN_PATH) . DIRECTORY_SEPARATOR . 'themes'],
+            ]);
+
+            (new RtlStylesheetProcessor(
+                $this->container->getProperty(UpgradeContainer::PS_ADMIN_PATH),
+                $this->container->getProperty(UpgradeContainer::PS_ROOT_PATH) . DIRECTORY_SEPARATOR . 'themes',
+                []
+            ))
+                ->setProcessBOTheme(true)
+                ->setProcessDefaultModules(true)
+                ->process();
+        }
+
+        // FO themes
+        if (!class_exists(AdaptThemeToRTLLanguagesCommand::class)) {
+            return;
+        }
+
+        $this->logger->info($this->container->getTranslator()->trans('Upgrade the RTL files of front-office themes.', [], 'Modules.Autoupgrade.Admin'));
```

Comment:
```suggestion
        $this->logger->info($this->container->getTranslator()->trans('Upgrade the RTL files of front-office themes.'));
```

Reply:
This will be done by another PR, I wait for it to be merged first :smirk: